packages/utils/src/git-utils.ts

```diff
@@ -5,9 +5,14 @@ export async function checkIsFileGitIgnored(path: string, warnIfNotGitRepo = fal
     await asyncExec(`git check-ignore ${path} -q`);
     return true;
   } catch (err) {
+    const stderr = (err as any).stderr as string;
+    // git is not installed, so we don't know
+    if (stderr.includes('command not found')) return undefined;
```

Comment:
main difference here is that we are returning undefined to signify that we don't know. That difference between false and undefined is not used anywhere yet, but it feels a bit safer for the future.